pbm/storage/s3/s3.go

```diff
@@ -550,6 +558,20 @@ func (s *S3) session() (*session.Session, error) {
 		}})
 	}
 
+	// If using GCE, attempt to retrieve access token from metadata server
+	if onGCE() {
+		tokenSource := google.ComputeTokenSource(s.opts.ServiceAccount, "")
+		token, err := tokenSource.Token()
+		if err != nil {
+			return nil, errors.Wrap(err, "get GCP token")
+		}
+		providers = append(providers, &credentials.StaticProvider{Value: credentials.Value{
+			AccessKeyID:     "GCP_OAUTH_TOKEN",
+			SecretAccessKey: "GCP_OATH_TOKEN",
```

Comment:
is `GCP_OATH_TOKEN` a typo?

Reply:
Sorry for the delay, no not a typo, the fields are mandatory but they're just set to placeholder values here since they're not used for GCS.